scss/components/_breadcrumb.scss

```diff
@@ -41,4 +41,44 @@
 	&__link {
 		//
 	}
+}
+
+.breadcrumb-neutral {
+	padding: ($baseline * 2) 0 ($baseline * 1) 0;
+
+	@include breakpoint(sm) {
+		padding: $baseline 0 ($baseline * 2) 0;
+	}
+
+
+	&__list {
+		list-style: none;
+		margin: 0;
+		padding: 0;
+	}
+
+	&__item {
+		display: inline-block;
+		margin: 0 8px 0 0;
+		padding: 6px 0 10px 0;
+
+		@include breakpoint(sm) {
+			line-height: 16px;
+			padding: 10px 0 6px 0;
+		}
+
+		a {
+			text-decoration: underline;
+		}
+
+		&:after {
+			content: '>';
+			padding: 0px 0 0 8px;
```

Comment:
We probably only need to set `padding-left: 8px;` here. The main breadcrumb class could probably have the same change too.